contrib/test_jenkins.sh

```diff
@@ -682,11 +682,17 @@ run_hello() {
 	test_args="$@"
 	test_name=${api}_hello_world
 
+	extra_gcc_options=""
+	if module_load dev/cuda && [ "X$have_cuda" == "Xyes" ]
+	then
+		extra_gcc_options="-lcuda -lcudart -DUCX_HELLO_WORLD_CUDA"
+	fi
+
 	if [ ! -x ${test_name} ]
 	then
 		gcc -o ${test_name} ${ucx_inst}/share/ucx/examples/${test_name}.c \
 		    -l${api} -lucs -I${ucx_inst}/include -L${ucx_inst}/lib \
-		    -Wl,-rpath=${ucx_inst}/lib
+		    -Wl,-rpath=${ucx_inst}/lib $extra_gcc_options
```

Comment:
@hoopoepg added option to compile hello word tests as part of configure-devel
so let's use that here as well, and add "xx_CPPFLAGS = -DHAVE_CUDA" to Makefile.am of the examples